File: app/src/main/java/me/echeung/moemoekyun/client/api/library/Jpop.java
package me.echeung.moemoekyun.client.api.library;

public class Jpop extends Library {

    public static final Library INSTANCE = new Jpop();

    public static final String NAME = "jpop";

    private Jpop() {
        super(NAME, "wss://listen.moe/gateway", "https://listen.moe/fallback");
    }

}

File: app/src/main/java/me/echeung/moemoekyun/client/api/library/Jpop.kt
package me.echeung.moemoekyun.client.api.library

class Jpop private constructor() : Library(
    NAME,
    "wss://listen.moe/gateway_v2",
    "https://listen.moe/fallback"
) {
    companion object {
        const val NAME = "jpop"

        val INSTANCE: Library = Jpop()
    }
}
